lib/components/viewers/stop-viewer.js

```diff
@@ -185,16 +186,22 @@ class StopViewer extends Component {
       // (e.g. cases where a user in New York looks at a schedule in Los Angeles).
       timezoneWarning = (
         <StyledAlert bsStyle='info'>
-          <Glyphicon glyph='info-sign' /> Departure times
-          are shown in <strong>{timezoneCode}</strong>.
+          <Glyphicon glyph='info-sign' />
```

Comment:
There was a space that is now lost. Is there an equivalent icon name to use with `Icon`?

Reply:
Yep, `Icon type='info-circle'` works and looks good here. Changed in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/2ccefb263ad8863edab72e9d796596beda7336f0